client/API.md

```diff
@@ -0,0 +1,536 @@
+# Kapacitor API Reference Documentation
```

Comment:
So I forgot to mention this. The API also has the `/write` endpoint that behaves just like the InfluxDB `/write` endpoint. Since the InfluxDB API is not versioned most clients expect to hit `/write` and call it a day. I originally has the API versioned but found that most clients couldn't talk to Kapacitor as a result since they didn't allow the URL path to be changed, so I removed it. 

But since its only the `/write` path that has the issue maybe we version the whole API and then add an alias from `/write` to `/v1/write` for these older clients?

>Additionally, when one tries to sew many APIs routes into a single domain it is nice if they are namespaced.

Wouldn't that be the job of the sewer (aka load balancer)? Seem's like if your are doing this you have to use something to proxy the requests and whatever that is, it would  be trivial to add a path rewrite step.